src/fidgets/ui/Text.tsx

```diff
@@ -26,33 +26,44 @@ export type TextFidgetSettings = {
   headingsFontColor?: string;
 } & FidgetSettingsStyle;
 
+const defaultText = `Add formatted text, links, images, or even code blocks with the Text Fidget. To format your text or embed content such as images, use Markdown Syntax.`;
+
+
 export const textConfig: FidgetProperties = {
   fidgetName: "Text",
   icon: 0x1f4c4,
   fields: [
     {
       fieldName: "title",
+      displayName: "Title",
+      displayNameHint: "Optional title for your Text Fidget.",
       default: "Text Fidget",
       required: false,
       inputSelector: TextInput,
       group: "settings",
     },
     {
       fieldName: "text",
-      default: "Jot down your ideas and grow them.",
+      displayName: "Text",
+      displayNameHint: "Input text to display and use Markdown Syntax to format or embed content like images.",
+      default: defaultText,
       required: true,
       inputSelector: CSSInput,
       group: "settings",
     },
     {
       fieldName: "fontFamily",
+      displayName: "FontFamily",
```

Comment:
thanks for adding displayNames to these fields 🙏  while we're at it, please format any displayNames to use correct human syntax if they aren't already, ie. Font Family instead of FontFamily.